src/LuaPushPull.h

```diff
@@ -29,6 +29,18 @@ inline void pi_lua_generic_pull(lua_State * l, int index, std::string & out) {
 	const char *buf = luaL_checklstring(l, index, &len);
 	std::string(buf, len).swap(out);
 }
+template <typename Type>
+inline void LuaPush(lua_State *l, Type value) {
```

Comment:
Oh, yeah, that's a good idea! Although I would have kept it lower case, since it's a mere function and not a method nor a type. But that's just me, you can ignore me.